Analysis/Core/include/AnalysisCore/RecoDecay.h

```diff
@@ -707,7 +707,7 @@ class RecoDecay
         }
         // Check that the number of direct daughters is not larger than the number of expected final daughters.
         if (indexDaughterFirst > -1 && indexDaughterLast > -1 && indexDaughterLast - indexDaughterFirst + 1 > N) {
-          //Printf("MC Rec: Rejected: too many direct daughters: %d (expected %d final)", indexDaughterLast - indexDaughterFirst + 1, N);
+          //Printf("MC Rec: Rejected: too many direct daughters: %d (expected %ld final)", indexDaughterLast - indexDaughterFirst + 1, N);
```

Comment:
What is the motivation of these changes?

Reply:
Compilation warnings when uncommenting the lines.